src/chapters/lists/exercises.rst

```diff
@@ -116,8 +116,8 @@ or `Trinket <https://trinket.io/python/b010aba314?showInstructions=true>`__.
 Part Four: Multi-dimensional Lists
 ----------------------------------
 
-*Part 4 starter code*: `repl.it <https://repl.it/@launchcode/LCHS-List-Exercises-Part-4#main.py>`__
-or `Trinket <https://trinket.io/python/f7573d922f?showInstructions=true>`__.
+*Part 4 starter code*: `repl.it <https://replit.com/@launchcode/LCHS-List-Exercises-Part-4-1#main.py>`__
+or `Trinket <https://trinket.io/python/93c5ca4e97?showInstructions=true>`__.
```

Comment:
I prefer the original main.py  with the more detailed #comments found here: https://replit.com/@launchcode/LCHS-List-Exercises-Part-4#main.py, the link when I click on the textbook as it is now.,  I would copy and add that to the trinket.  Having the read me contain the instructions is good.

Reply:
The `instructions.md` file in the new Replit was inspired by the Trinket editors, which display an Instructions tab when the link is opened.  When I made the original starter code, Trinket let me include detailed steps, while Replit only seemed to support `# Comments`.  That caused the different editor contents in the original starter code samples.

Do you want to shift to comments only in both editors, use minimal comments combined with detailed instructions in the separate tab/file, or keep both the detailed comments and the separate tab/file?

Reply:
I like the more detailed comments in the starter code, but you make a good point about how in longer assignments this could be problematic.  This could be a good way to get students in the habit of  adding their own comments.  Maybe a prompt at the top something like this?  (#Remember to add comments to explain and make your code more readable to others)

Reply:
Got it.  I'll update the comments to prompt the kids to add comments!  ;-)